jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -628,6 +663,39 @@
 					"response": []
 				},
 				{
+					"name": "create jvm2 for duplicate error check",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 201\"] = responseCode.code === 201;",
```

Comment:
Same thing: the name of the test is "duplicate error check" but the expected response is 201 and the ID is being saved as an environment variable

Reply:
I created this group to check "update to existing name exception" same for jvm webserver and webapp